addons/l10n_pk/demo/account_demo.py

```diff
@@ -0,0 +1,379 @@
+                            'quantity': 3,
+                            'price_unit': 2000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_services_10')
+                            ])],
+                        }),
+                    ]
+                },
+                'demo_vendor_bill_b2b_3': {
+                    'ref': 'INV/003',
+                    'move_type': 'in_invoice',
+                    'partner_id': 'res_partner_b2b',
+                    'invoice_user_id': 'base.user_demo',
+                    'invoice_payment_term_id': 'account.account_payment_term_end_following_month',
+                    'invoice_date': datetime.now(),
+                    'invoice_line_ids': [
+                        Command.create({
+                            'product_id': 'product.product_product_9',
+                            'quantity': 1,
+                            'price_unit': 1000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_services_17')
+                            ])],
+                        }),
+                        Command.create({
+                            'product_id': 'product.product_product_10',
+                            'quantity': 1,
+                            'price_unit': 2000.0,
+                            'tax_ids': [Command.set([
+                                _get_tax_by_id('pk_sales_tax_services_10'),
+                                _get_tax_by_id('pk_sales_tax_services_1')
+                            ])],
+                        }),
+                    ]
+                },
+            }
+        else:
+            return super()._get_demo_data_move(company)
+
+    @api.model
+    def _get_demo_data_config_settings(self, company=False):
+        return {
+            'sales_credit_limit': {
+                'account_use_credit_limit': True,
+                'account_default_credit_limit': '10000'
+            }
+        }
+
+    def _post_load_demo_data(self, company=False):
+        if company.account_fiscal_country_id.code == "PK":
+            if company.state_id:
+                invoices = (
+                    self.ref('demo_invoice_b2b_1')
+                    + self.ref('demo_invoice_b2b_2')
+                    + self.ref('demo_invoice_b2c_1')
+                    + self.ref('demo_invoice_b2c_2')
+                    + self.ref('demo_reconcile_b2b_1')
+                    + self.ref('demo_vendor_bill_b2b_1')
+                    + self.ref('demo_vendor_bill_b2b_2')
+                    + self.ref('demo_vendor_bill_b2b_3')
+                )
+                for move in invoices:
+                    try:
+                        move.action_post()
+                    except (UserError, ValidationError):
+                        _logger.exception('Error while posting demo data')
```

Comment:
Why?  Just let it traceback.

Reply:
https://github.com/odoo/odoo/blob/2be7f413493a6ad43980eb031b8383deb3a706c0/addons/account/demo/account_demo.py#L59
https://github.com/odoo/odoo/blob/2be7f413493a6ad43980eb031b8383deb3a706c0/addons/l10n_in/demo/account_demo.py#L610

account and l10n_in use the same. is it fine to remove it?